stubs/django/contrib/auth/models.pyi

```diff
@@ -19,4 +19,10 @@ class AbstractUser(AbstractBaseUser):
 class User(AbstractUser):
     pass
 
+class BaseUserManager(models.Manager):
```

Comment:
It looks like this actually lives in `base_user.py`:
https://github.com/django/django/blob/e29637681be07606674cdccb47d1e53acb930f5b/django/contrib/auth/base_user.py#L16

Would you mind putting it there and importing it in this file instead?

Reply:
It does, but so does [`AbstractBaseUser`](https://github.com/django/django/blob/e29637681be07606674cdccb47d1e53acb930f5b/django/contrib/auth/base_user.py#L47). That's why I put it here. Want me to move both?

Reply:
Yeah, if you could that'd be great. These models have sometimes been evolved quickly to prevent typechecking errors, without looking at the original source. I guess in this case, everyone imported `AbstractBaseUser` from `django.contrib.auth.models`, so that's where we added the placeholder class.